README.md

```diff
@@ -193,9 +193,10 @@ Advanced usage:
     {outputName}      The output file's filename without extension
     {outputExt}       The output file's extension
 
-    {pocket}  The ROM's core-specific /Assets/* directory for the Analogue Pocket (e.g. "gb")
-    {mister}  The ROM's core-specific /games/* directory for the MiSTer FPGA (e.g. "Gameboy")
-    {onion}   The ROM's emulator-specific /Roms/* directory for OnionOS/GarlicOS (e.g. "GB")
+    {pocket}    The ROM's core-specific /Assets/* directory for the Analogue Pocket (e.g. "gb")
+    {mister}    The ROM's core-specific /games/* directory for the MiSTer FPGA (e.g. "Gameboy")
+    {onion}     The ROM's emulator-specific /Roms/* directory for OnionOS/GarlicOS (e.g. "GB")
+    {batocera}  The ROM's emulator-specific /roms/* directory for Batocera (e.g. "gb")
```

Comment:
@dcramer copying the comment here so we can thread a discussion:

> fwiw new to both Igir and Batocera so possible I'm ignorant of something. Primary motivation here is to avoid a bunch of roughly duplicate directories (e.g. NES Headered + NES Headerless) which would then need collapsed vs simply renamed. Othewrise I would probably have just bash'd this out

Interesting, I didn't realize Batocera had custom directories, I figured it just behaved like RetroArch. I have some new Batocera docs in https://github.com/emmercm/igir/pull/550 that will be published in the next version, but it looks like you already found it. I'll have to play with Batocera myself to see what the experience is like.

For avoiding some DATs, there's a `--dat-regex-exclude` option that you can use to filter out headerless DATs and the like without needing to unzip the DATs.

Reply:
Yeah I dont understand the ecosystem to understand the diff between batocera/retroarch/emulationstation/etc, but as far as I could tell its explicit in the directory naming for batoceras implementation (it will recreate the directories if you remove them as an example), and the way you structured the project it seemed the intended goal was to make it easy to repeatedly sync/update a library (which is def my goal!)